src/styles/functions.scss

```diff
@@ -0,0 +1,3 @@
+@function getDaysWidth($days-block-width) {
+  @return 8 * $days-block-width;
```

Comment:
What is 8? Your calculation is wrong. You need to add gaps to it.
Also, accept all numbers as arguments.

![Screenshot 2023-02-07 at 18 13 01](https://user-images.githubusercontent.com/54940595/217300507-eae409ce-e1de-40e6-a320-8a8019a71ff4.png)